### PR TITLE
WS-3069: add lead_capture_form_url field to Degree model

### PR DESCRIPTION
**Pull request by danielkalen**
[WS-3069](https://2u-internal.atlassian.net/browse/WS-3069) requests that editors would be able to specify a custom url for a form to be rendered in an iframe in place of the current hubspot form on masters pages. An example page with that form in the sidebar: https://www.edx.org/masters/online-master-business-administration-mba-bux.

In order for this feature to be provided, a new field had to be created for the Degree model.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @danielkalen!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.


:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://openedx.org/cla)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.     If you've signed an agreement in the past, you may need to re-sign.  See     [The New Home of the Open edX Codebase](https://open.edx.org/blog/the-new-home-of-the-open-edx-codebase/)     for details.

**Review by marekwrobel — APPROVED** [review]

File: course_discovery/apps/course_metadata/models.py [hunk 1]
```diff
@@ -2853,7 +2853,11 @@ class Degree(Program):
         blank=True,
         max_length=128,
     )
-
+    lead_capture_form_url = models.URLField(
```
Comment (DawoudSheraz) [inline course_discovery/apps/course_metadata/models.py]: Verify this field in test_models as well.

**Review by DawoudSheraz — APPROVED** [review]
I will merge this once CI is green. Please squash your commits and rebase your branch.

**Comment by natabene** [comment]
@DawoudSheraz Please wait for CLA check to go green, too.

**Comment by natabene** [comment]
@danielkalen You have been properly added to the CLA list, so all your future PRs won't be flagged as external. Sorry for the trouble. If you see red CLA check in your other PRs you already opened, you can edit the title and save. That will trigger the check to re-run and it will turn green.
Can you also do me a favor and remove the open-source-contribution label? I don't have the permission to do so myself. Thanks!

**Comment by mphilbrick211** [comment]
@danielkalen Just checking in on this to see if you are planning to re-run the failing tests on this PR?

**Comment by danielkalen** [comment]
Yes, right after I push my commits (tmw afternoon EST)

**Comment by mphilbrick211** [comment]
Hi @danielkalen - just checking back in on this. Please let me know if you plan to keep this PR open. Thanks!

**Comment by danielkalen** [comment]
Hi Michelle I got caught up with something else that kept me busy but I’ll
be able to return and work on what dawoud asked.

**Comment by mphilbrick211** [comment]
Hi @danielkalen! Friendly ping on this to see if you are still planning to proceed?

**Comment by mphilbrick211** [comment]
Hi @danielkalen, me again :) Just checking in on this PR. Let me know if you plan to pursue. Thanks!

**Comment by mphilbrick211** [comment]
Another friendly ping on this @danielkalen :)

**Comment by danielkalen** [comment]
Hey Michelle, so sorry I missed your other pings. This ticket ended up
being scratched all together. Apologies for not letting you know earlier!

**Comment by danielkalen** [comment]
Actually, one sec, I think I confused this with another ticket. Let me
check again tonight and I’ll get back to you on the final decision
regarding this.

**Comment by danielkalen** [comment]
Yes, seems like I got caught up with other tickets and let this fly under the radar. I'll work on getting the CI to be green tmw and ask for a reapproval from the Phoenix team once done. Once again apologies for the delay.

**Comment by danielkalen** [comment]
Closed because this has already been taken care of by another PR. Sorry for the mixups - confusions upon confusions

**Pull request closed without merging**